### ISSUE TITLE
Pre-encoding while waiting

### ISSUE THREAD

**Issue by kc7zzv** [keredson/gnomecast#115]
It would be nice if the next video could render while we watch the first one, and if the first video would render while I tinker with connection settings, and get my chromecast running.

Request:
Please start encoding the next video at low priority, before the user pushes play.

This won't be as important once #42 is done, but should still compliment it.

**Comment by ntc2** [comment]
I just started using Gnomecast and this is the feature I'm missing. I've been watching TV shows and I'd like to transcode the next episode while I'm watching the current episode. A simple implementation could be to transcode all files that the user queues up, as soon as they're queued.

**Comment by keredson** [comment]
Transcoding everything up front runs the risk of eating through all available disk.

Earlier I had code that would just transcode the next video while the current video way playing.  I think I removed it when I was trying to hunt down the dup transcoding instances.  (Still not fully solved.). But yes, agree a good feature.
